pkg/scheduler/plugins/rescheduling/low_node_utilization.go

```diff
@@ -59,13 +59,29 @@ func (lnuc *LowNodeUtilizationConf) parse(configs map[string]interface{}) {
 	}
 	lowThresholdsConfigs, ok := configs["thresholds"]
 	if ok {
-		lowConfigs, _ := lowThresholdsConfigs.(map[string]int)
-		parseThreshold(lowConfigs, lnuc, "Thresholds")
+		lowConfigs, ok := lowThresholdsConfigs.(map[interface{}]interface{})
+		if !ok {
+			klog.Errorln("convert lowConfigs error")
```

Comment:
Volcano log is a complete sentence. Please update it.

Reply:
OK.